bin/brain-even.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 
 import brainEven from '../src/Games/brain-even.js';
+import theCoreOfGames from '../src/Games/index.js';
```

Comment:
Plz don't use 'the' in var names. Maybe gamesCore or gameEngine?

Reply:
fixed